src/svea_sensors/launch/localize.launch

```diff
@@ -6,74 +6,172 @@
+    and actuation_to_twist working) -->
+    <arg name="start_serial"            default="false" />
+    <arg name="device"                  default="/dev/GPS" />
+    <arg name="delay"                   default="2.0"/>
+    <!-- yaw_offset is set to 0 if the robot is facing East during initialization -->
+    <arg name="yaw_offset"              default="0"/>
+    <arg name="port"                    default="/dev/ttyACM0"/>
+
+    <!-- For camera and aruco detection-->
+    <arg name="use_web_camera"          default="false"/>
+    <arg name="camera_info_file"        default="usbc_web_camera.yaml"/>
+    <arg name="image"                   default="/camera/image_raw"/>
+    <arg name="aruco_pose"              default="aruco_pose"/>
+    <arg name="aruco_dict"              default="DICT_4X4_250"/>
+    <arg name="aruco_size"              default="0.365"/>
+    <arg name="aruco_tf_name"           default="aruco"/>
+    <arg name="device_id"               default="1"/> 
+    <arg name="static_gps_fix_topic"    default="static/gps/fix"/>
+    <arg name="odometry_gps_topic"      default="static/odometry/gps"/>
+    <arg name="aruco_pose_topic"        default="aruco_pose"/>
+    <arg name="aruco_id"                default="11"/>    
+    <arg name="pedestrian_demo"         default="false"/>
 
     <!-- Start the stereo camera -->
     <group if="$(arg use_camera)">
         <param name="use_sim_time" value="false"/>
         <include if="$(eval camera == 'rs')" file="$(find svea_sensors)/launch/rs_odometry.launch">
-            <arg name="xavier" value="$(arg xavier)"/>
+            <arg name="xavier"         value="$(arg xavier)"/>
+            <arg name="is_indoors"     value="$(arg is_indoors)"/>
+            <arg name="initial_pose_x" value="$(arg initial_pose_x)" />
+            <arg name="initial_pose_y" value="$(arg initial_pose_y)" />
+            <arg name="initial_pose_a" value="$(arg initial_pose_a)" />
+            <arg name="pedestrian_demo" value="$(arg pedestrian_demo)"/>
+            
         </include>
         <include if="$(eval camera == 'zed')" file="$(find svea_sensors)/launch/zed_odometry.launch">
-            <arg name="xavier" value="$(arg xavier)"/>
+            <arg name="xavier"  value="$(arg xavier)"/>
         </include>
     </group>
 
+    <!-- Start low-level interface -->
+    <group if="$(arg start_serial)">
+        <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
+            <param name="port" value="$(arg port)"/>
+            <param name="baud" value="250000" />
+        </node>
+    </group>
+
     <!-- Start wheel encoders -->
     <group if="$(arg use_wheel_encoders)">
-        <include file="$(find svea_sensors)/launch/wheel_odometry.launch">
-            <arg name="start_serial" value="false"/>
-        </include>
+        <include file="$(find svea_sensors)/launch/wheel_odometry.launch" />
```

Comment:
wheel_odometry doesn't start serial by default right?

Reply:
It does not.